internal/weldr/api.go

```diff
@@ -1280,6 +1281,68 @@ func (api *API) composeHandler(writer http.ResponseWriter, request *http.Request
 	json.NewEncoder(writer).Encode(reply)
 }
 
+func (api *API) composeCancelHandler(writer http.ResponseWriter, request *http.Request, params httprouter.Params) {
+	if !verifyRequestVersion(writer, params, 0) {
+		return
+	}
+
+	type composeCancelStatus struct {
+		UUID   uuid.UUID `json:"uuid"`
+		Status bool      `json:"status"`
+	}
+
+	type composeCancelError struct {
+		ID  string `json:"id"`
+		Msg string `json:"msg"`
+	}
+
+	uuidString := params.ByName("uuid")
+
+	errors := []composeCancelError{}
```

Comment:
Do we need to use array of errors when we can have maximum of 1 error?

Reply:
We could make it a pointer instead. Probably should :)